packages/knip/src/plugins/biome/index.ts

```diff
@@ -0,0 +1,49 @@
+import type { IsPluginEnabled, Plugin, PluginOptions, ResolveConfig } from '../../types/config.js';
+import { toDeferResolve, toDependency, toIgnore, type ConfigInput, type Input } from '../../util/input.js';
+import { hasDependency } from '../../util/plugin.js';
+import type { BiomeConfig } from './types.js';
+
+const title = 'biome';
+
+const enablers = ['@biomejs/biome'];
+
+const isEnabled: IsPluginEnabled = ({ dependencies }) => 
+  {
+    const enabled = hasDependency(dependencies, enablers);
+    return enabled;
+  }
+
+const config: string[] = ['biome.json'];
+
+
+const resolveExtends = (extendsArray: string[], options: PluginOptions): Input[] => {
+  const inputs = [] as Input[];
+  for (let item of extendsArray) { 
+    // https://biomejs.dev/guides/configure-biome/#share-a-configuration-file
+    if (item.endsWith(".json")) {
+      inputs.push(toDeferResolve(item));
+    } else {
+      if (require.resolve(item, { paths: [options.cwd] })) {
```

Comment:
Ideally, plugins aren't concerned with doing any resolving like this themselves

Reply:
Hmm what is your recommendation to support module-based `extends` in biome config?

https://biomejs.dev/guides/configure-biome/#share-a-configuration-file

Reply:
We can use `toConfig()` for this (https://knip.dev/guides/writing-a-plugin#toconfig, while I realize the docs say "by a different plugin" this could also be another config file but the same plugin)

Reply:
The TypeScript plugin does basically the same thing: https://github.com/webpro-nl/knip/blob/18842738a4acc06e11654a853c5247f0e8399d27/packages/knip/src/plugins/typescript/index.ts#L22-L26